example.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
```

Comment:
I know we are using `requests` library on `crash_test.py`, but I would like to suggest to have a dependency-free `example.py`. We could add complex cases (with external dependencies) in a test script. What do you think?